lib.php

```diff
@@ -966,28 +933,27 @@ function block_coursefeedback_answers_exist($feedbackid) {
  */
 function block_coursefeedback_validate($feedbackid, $returnerrors = false)
 {
-	$notifications = array();
-	$feedbackid    = intval($feedbackid);
-	if ($feedbackid > 0)
-	{
-		$langs = block_coursefeedback_get_combined_languages($feedbackid);
-		if (empty($langs))
-			$notifications[] = get_string("page_html_norelations", "block_coursefeedback");
-		$count = block_coursefeedback_get_questionid($feedbackid) - 1;
-		if ($count !== count(block_coursefeedback_get_questions($feedbackid)))
-			$notifications[] = get_string("page_html_servedefaultlang",
-			                              "block_coursefeedback",
-					                      get_config("block_coursefeedback", "default_language"));
-	}
-	if ($returnerrors)
-		return $notifications;
-	else
-		return empty($notifications);
+    $notifications = array();
+    $feedbackid = intval($feedbackid);
+    if ($feedbackid > 0) {
+        $langs = block_coursefeedback_get_combined_languages($feedbackid);
+        if (empty($langs))
+            $notifications[] = get_string("page_html_norelations", "block_coursefeedback");
```

Comment:
Wenn wir es eh schon anfassen... der Moodle Coding Style ist immer geschweifte Klammern. D.h. diese Einzeiler müssen sollten ebenfalls dementsprechend geändert werden.

Reply:
```suggestion
        if (empty($langs)) {
            $notifications[] = get_string("page_html_norelations", "block_coursefeedback");
        }
```

Reply:
Ja hab das auf dem Schirm (Martin hatte das auch schon angemerkt), diese Coding style Änderungen würde ich dann alle in einen Commit und PR packen ;)

Reply:
Dann kann ich diesen PR ja schließen... nicht wahr?